dart/lib/src/protocol/sentry_span.dart

```diff
@@ -32,7 +32,7 @@ class SentrySpan extends ISentrySpan {
     this.samplingDecision,
     Function({DateTime? endTimestamp})? finishedCallback,
   }) {
-    _startTimestamp = startTimestamp?.toUtc() ?? getUtcDateTime();
+    _startTimestamp = (startTimestamp ?? _hub.options.clock()).toUtc();
```

Comment:
`_hub.options.clock()` already returns UTC, you are calling it again.

Reply:
Yep that makes sense, but this can be:

`startTimestamp?.toUtc() ?? _hub.options.clock()`
Just a micro-optimization